utils/src/main/java/com/cloud/utils/UriUtils.java

```diff
@@ -23,7 +23,6 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.HttpURLConnection;
-import java.net.Inet6Address;
```

Comment:
@wido I think this needs to be removed - looks like you've turned off maintainers to be able to make edits.